website/docs/maintenance/observability/monitor-metrics.md

```diff
@@ -19,797 +19,297 @@ sidebar_position: 3
+|---------|-----------------------|-------------------------------|-------|
+| `request` | `bytesInPerSecond`  | Bytes received per second.    | Gauge |
+|          | `bytesOutPerSecond`  | Bytes sent per second.        | Meter |
+|          | `requestsPerSecond`  | Requests per second.          | Meter |
+|          | `responsesPerSecond` | Responses per second.         | Meter |
+|          | `requestLatencyMs`   | Request latency.              | Gauge |
+|          | `requestsInFlight`   | In-flight requests.           | Gauge |
+
+---
+
+### Table Metrics 
+#### Scope: TabletServer, `table` infix
+
+| Metric                                    | Description                                   | Type  |
+|-------------------------------------------|-----------------------------------------------|-------|
+| `messagesInPerSecond`                     | Messages written per second.                  | Meter |
+| `bytesInPerSecond`                        | Bytes written per second.                     | Meter |
+| `bytesOutPerSecond`                       | Bytes read per second.                        | Meter |
+| `totalProduceLogRequestsPerSecond`        | Produce-log requests per second.              | Meter |
+| `failedProduceLogRequestsPerSecond`       | Failed produce-log requests per second.       | Meter |
+| `totalFetchLogRequestsPerSecond`          | Fetch-log requests per second.                | Meter |
+| `failedFetchLogRequestsPerSecond`         | Failed fetch-log requests per second.         | Meter |
+| `totalPutKvRequestsPerSecond`             | PUT-kv requests per second.                   | Meter |
+| `failedPutKvRequestsPerSecond`            | Failed PUT-kv requests per second.            | Meter |
+| `totalLookupRequestsPerSecond`            | Lookup requests per second.                   | Meter |
+| `failedLookupRequestsPerSecond`           | Failed lookup requests per second.            | Meter |
+| `totalLimitScanRequestsPerSecond`         | Limit-scan requests per second.               | Meter |
+| `failedLimitScanRequestsPerSecond`        | Failed limit-scan requests per second.        | Meter |
+| `totalPrefixLookupRequestsPerSecond`      | Prefix-lookup requests per second.            | Meter |
+| `failedPrefixLookupRequestsPerSecond`     | Failed prefix-lookup requests per second.     | Meter |
+| `remoteLogCopyBytesPerSecond`             | Bytes copied to remote per second.            | Meter |
+| `remoteLogCopyRequestsPerSecond`          | Remote log-copy requests per second.          | Meter |
+| `remoteLogCopyErrorPerSecond`             | Failed remote log-copy requests per second.   | Meter |
+| `remoteLogDeleteRequestsPerSecond`        | Remote log-delete requests per second.        | Meter |
+| `remoteLogDeleteErrorPerSecond`           | Failed remote log-delete requests per second. | Meter |
+
+---
+
+### Bucket Metrics 
+#### Scope:`table_bucket` infix
```

Comment:
![image](https://github.com/user-attachments/assets/0aaabcf1-caa0-4788-a21d-6ddc6accdef7)
The content `Scope:`table_bucket` infix` looks a bit strange. If the infix is empty, do we still need to include the title? And if we do need to include it, should we use a different format?